modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowTaskManagerImplTest.java

```diff
@@ -825,6 +830,54 @@ public void testDeleteUserWithWorkflowTaskAssigned() throws Exception {
 		_deactivateWorkflow(BlogsEntry.class.getName(), 0, 0);
 	}
 
+	@Test
+	public void testFailedScriptedAssignment() throws Exception {
+		String workflowDefinitionName = "Failed Scripted Single Approver";
+
+		_createScriptedAssignmentWorkflow(
+			"single-approver-failed-scripted-assignment-workflow-definition." +
+				"xml",
```

Comment:
rename to `broken-scripted-assignment-workflow-definition.json`

Reply:
tip: you can leverage workflow admin API performing GET operation to retrieve your workflow definition content as json.

Reply:
Done!